chat/uikit/ios.md

```diff
@@ -0,0 +1,386 @@
+then the title bar displays participants' usernames (excluding the current user) in the title bar.
+
+```obj-c
+[SKYChatConversationView UICustomization].titleDisplayType = SKYChatConversationViewTitleOptionsOtherParticipants;
+```
+
+```swift
+SKYChatConversationView.UICustomization().titleDisplayType = .otherParticipants
+```
+
+
+### Message
+|`attribute`|description|type|
+|---------|-----------|------|
+|`messageSenderTextColor`|text color of the message sender|Color|
+
+
+## SKYChatUIModelCustomization
+You can change `userNameField` and `userAvatarField` via `SKYChatUIModelCustomization.default()`.
+
+
+```obj-c
+[[[SKYChatUIModelCustomization default]
+    updateWithUserNameField:@"name"]
+    updateWithUserAvatarField:@"profile_pic"];
+```
+
+```swift
+SKYChatUIModelCustomization.default()
+    .update(userNameField: "name")
+    .update(userAvatarField: "profile_pic")
+```
+
+|Attribute|`Description`|Type|
+|---------|-----------|------|
+|`userNameField`|Field name of the username in `user` table|String, default "username"|
+|`userAvatarField`|Field name of the avatar image in `user` table|String, default "avatar"|
+
+
+##### `SKYChatConversationViewController`''s properties
+
+You may also customize `SKYChatConversationViewController`s properties before it is displayed.
+
+```obj-c
+SKYChatConversationViewController* conversationViewController = [[SKYChatConversationViewController alloc] init];
+conversationViewController.conversation = c;
+conversationViewController.incomingMessageBubbleColor = [UIColor redColor];
+[self.navigationController pushViewController:conversationViewController animated:YES];
+```
+
+```swift
+let conversationViewController = SKYChatConversationViewController()
+conversationViewController.converation = c
+conversationViewController.incomingMessageBubbleColor = UIColor.red
+self.navigationController?.pushViewController(conversationViewController, animated: true)
+```
+
+
+##### Message
+|`attribute`|description|type|
+|---------|-----------|------|
+|`incomingMessageBubbleColor`|Incoming messages background color|Color|
+|`outgoingMessageBubbleColor`|Outgoing messages background color|Color|
+
+
+##### Feature On/Off
+|Attribute|`Description`|Type|
+|---------|-----------|------|
+|`shouldShowCameraButton`|Show camera button if true|Bool, default: True|
+|`shouldShowVoiceMessageButton`|Show voice message button if true|Bool, default: True|
+|`shouldShowTypingIndicator`|Show typing indicators if true|Bool, default: True|
+|`shouldShowMessageStatus`|Show message status if true|Bool, default: true|
+|`shouldShowAccessoryButton`|Show accessory button if true|Bool, default: true|
+
+
+
+## Delegates
+You may also implement functions in `SKYChatConversationViewControllerDelegate`
+
+### Message Date
+
+- To customize date format
+```obj-c
+- (NSAttributedString *) conversationViewController:(SKYChatConversationViewController *)controller dateStringAt:(NSIndexPath *)indexPath
```

Comment:
Please put implementation example for those delegate functions which are not just return bool. That would be easier to understand :)

Reply:
updated.